Samples/Samples/ViewModel/CalendarEventAddViewModel.cs

```diff
@@ -330,13 +331,13 @@ public bool IsMonthDaySpecific
 
         public ObservableCollection<DayOfTheWeekSwitch> RecurrenceDays { get; set; } = new ObservableCollection<DayOfTheWeekSwitch>()
         {
-            new DayOfTheWeekSwitch() { Day = DayOfTheWeek.Monday },
-            new DayOfTheWeekSwitch() { Day = DayOfTheWeek.Tuesday },
-            new DayOfTheWeekSwitch() { Day = DayOfTheWeek.Wednesday },
-            new DayOfTheWeekSwitch() { Day = DayOfTheWeek.Thursday },
-            new DayOfTheWeekSwitch() { Day = DayOfTheWeek.Friday },
-            new DayOfTheWeekSwitch() { Day = DayOfTheWeek.Saturday },
-            new DayOfTheWeekSwitch() { Day = DayOfTheWeek.Sunday }
+            new DayOfTheWeekSwitch() { Day = DayOfTheWeek.Monday, IsChecked = false },
```

Comment:
Is the IsChecked=false necessary - wouldn't that be the default value

Reply:
Oh I see what's happened, an auto merge caused problems zz, this stuff wasn't meant to come through